recipe/build.sh

```diff
@@ -6,8 +6,25 @@ set -xe
 unset VERILATOR_ROOT
 
 autoconf
-./configure --prefix="$PREFIX"
+# as policy, conda-forge doesn't statically link any deps so --disable-partial-static
+# see commit message at https://github.com/verilator/verilator/commit/f00ff61559be0c6a5cbd07f25e264ce3e8652145
+# for details on what was being statically linked
+#
+# TODO consider adding --disable-defenv to configure to avoid baking the prefix into binaries
+#      removing the need for the C-string patch for relocatability.  Would need to set
+#      VERILATOR_ROOT and other env vars instead
+./configure --prefix="$PREFIX" \
+            --mandir="$PREFIX/man" \
+            --disable-partial-static
+
+# Set DTVERSION_rev to something more interesting than
+# "UNKNOWN_REV (mod)" since we're building from tar and it
+# won't ever show the git version
+echo "static const char* const DTVERSION_rev = \"conda-forge built ${PKG_BUILD_STRING}\";" > src/config_rev.h
```

Comment:
Oh, `PKG_BUILD_STRING` is set to `placeholder` here and that isn't what I want.  I was really hoping for it to have the hash and build number.  We may have to settle for just the build number.

Reply:
```suggestion
echo "static const char* const DTVERSION_rev = \"conda-forge build ${PKG_BUILDNUM}\";" > src/config_rev.h
```